src/utils.js

```diff
@@ -1,3 +1,5 @@
+const YT_BASE_URL = new URL('https://www.youtube.com/tv#/');
```

Comment:
We should probably use a plain URL string here, see below. (also we don't *really* use any `URL` features here outside of `.origin`)

Reply:
Nevermind.